device/mellanox/x86_64-nvidia_sn5600-r0/Mellanox-SN5600-O128/sai.profile

```diff
@@ -1 +1,2 @@
 SAI_INIT_CONFIG_FILE=/usr/share/sonic/hwsku/sai_sn5600_128x400g_1x25g.xml
```

Comment:
How does this work for other SKUs?

Reply:
We'd like to keep this flag for this specific SKU as there's a specific requirement to have this feature enabled there